packages/frameworks/vue/src/avatar/avatar.props.ts

```diff
@@ -1,16 +1,23 @@
-import type { Context } from '@zag-js/avatar'
+import type { Context, StatusChangeDetails } from '@zag-js/avatar'
 import type { PropType } from 'vue'
-import { declareEmits } from '../utils'
 
-export const props = {
+export const avatarProps = {
+  /** The document's text/writing direction. */
   dir: {
+    default: 'ltr',
```

Comment:
As far as I know, dir defaults to user's browser preference, so we shouldn't set this

Reply:
Reverted this, good point!